plugins/services/src/js/components/ServiceBreadcrumbs.js

```diff
@@ -25,6 +26,7 @@ function getHealthStatus(serviceID) {
   return (
     <div className="service-page-header-status page-header-breadcrumb-content-secondary muted">
       {`${serviceStatus} (${runningTasksCount} of ${instancesCount})`}
+      <ServiceStatusWarning item={service} showDebugInstruction={true} />
```

Comment:
I think the reason for adding the debug instruction in this case (when displayed in the breadcrumb) is because the debug tab is available to the user at this point, but would not be available when displayed in the services table.

@leemunroe What do you think?